common/src/main/java/com/datastax/oss/simulacron/common/cluster/QueryLog.java

```diff
@@ -160,6 +171,21 @@ public Frame getFrame() {
     return this.frame;
   }
 
+  /** @return List of decoded values from batch message if present and primed */
+  public List<LinkedHashMap<String, Object>> getDecodedValues() {
+    return this.decodedValues;
+  }
+
+  /** @return Decoded values from query/execute message if present and primed */
+  public LinkedHashMap<String, Object> getFirstDecodedValues() {
```

Comment:
De we need this one? It's a bit redundant with `getDecodedValues` and both are being included in the JSON payload:

```
        "firstDecodedValues" : {
          "a" : "any"
        },
        "decodedValues" : [ {
          "a" : "any"
        } ]
```

Reply:
agreed and removed